build.lua

```diff
@@ -18,7 +18,25 @@ scriptmanfiles = {"l3build.1"}
 sourcefiles  = {"*.dtx", "l3build*.lua", "*.ins"}
 typesetcmds  = "\\AtBeginDocument{\\DisableImplementation}"
 unpackdeps   = { }
-tagfiles     = {"l3build.1", "l3build.dtx", "*.md", "l3build.lua"}
+tagfiles     = {"l3build.1", "l3build.dtx", "*.md", "l3build.lua", "build.lua"}
+
+uploadconfig = {
+  version     = "2018-12-17",
```

Comment:
This works OK at present for development/experimental status, but I'd hope we can move to not having the data in the build script ... probably should have a function to extract it automatically from sources and/or Git/Subversion/... Something like `function get_tag_data()`?

Reply:
Indeed. One part of me thinks that if it's going to be hardcoded anywhere then the build.lua file should be it! I also like the idea of only "filling in" the data when `install` or `ctan` is run (but that only works for dates).

Like you say, this works for now...